ghost/tiers/lib/InMemoryTierRepository.js

```diff
@@ -17,6 +17,8 @@ class InMemoryTierRepository {
     toPrimitive(tier) {
         return {
             ...tier,
+            active: (tier.status === 'active'),
```

Comment:
This bit was confusing af to figure out. Basically anything that is a getter is not later accessible through JSON filtering. We have to do explicit mapping here to be able to filter by those values

Reply:
Damn okay - we can look at using the `toJSON` method of the objects here instead, which should make things easier